tutorial/20_recipes/010_reuse_best_trial.py

```diff
@@ -1,7 +1,7 @@
 """
 .. _reuse_best_trial:
 
-Re-use the best values
+Re-use the best parameters
```

Comment:
Personally, "best trial" is more accurate and consistent with the page's URL.
```suggestion
Re-use the best trial
```

Reply:
From my pov, `Re-use the best trial`  -> It will look good. If we only have the `study.best_trial`. But if we mention `study.best_trials` on the same page then `Re-use the best parameters` will be fine.

Reply:
> "best trial" is supposed to be more accurate

Sounds reasonable. Thank you for pointing it out.